Tasks/DownloadPackage/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "packagedownloader",
+  "name": "downloadpackage",
```

Comment:
can we add the package name to the instance name so that it appears as:

Download Package {Package name} in logs?